.github/ISSUE_TEMPLATE/gmentee_selection_template.md

```diff
@@ -0,0 +1,11 @@
+---
+name: To do list Issue
+about: Checklist to alert Gmentors to select their preferred Gmentee on the spreadsheet
+title: "[Gmentee Selection Checklist]"
+labels: to-do list, Gmentee selection 
+assignees: Network Managers
+---
+
+Notify them of newly accepted Gmnetees once in a month via:
```

Comment:
I did not remember we said once in a month. I thought we do on the fly or once a week